todel/src/models/logic/messages/edit.rs

```diff
@@ -114,7 +118,8 @@ impl Message {
             self.content = content;
         }
 
-        if edit.attachments.is_some() {
+        // does this change break anything?
+        if attachments.is_empty() {
```

Comment:
mate what the fuck

you made it only do shit if attachment it empty instead of if any exist

Reply:
there were 2 steps between that and what it ended up and somewhere along the line it was lost

Reply:
like you're still supposed to check if edit.attachments.is_some(), that makes the most sense, unchange it

Reply:
(didn't see this)
it can't be edit.attachments.is_some() because edit.attachments gets moved some lines above to create the *now replacing* !attachments.is_empty()

I could store this bool up there though if that makes more sense

Reply:
/shrug